molecule/default/converge.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    - default_mail_recipient: webmaster@acromedia.com
+  pre_tasks:
+
+    - name: Install nginx
+      apt:
```

Comment:
You'll need to either use "package" instead of "apt" (will only works if the package is the same name on both OS'es), or split this into OS-specific versions.